weldx/tests/test_utility.py

```diff
@@ -644,8 +679,5 @@ def test_coordinate_systems_modified(self):  # noqa: D102
 
 
 def test_is_interactive():
-    """Assert that the Pytest session is not recognized as interactive.
-
-    >>> assert not ut.is_interactive_session()
-    """
+    """Assert that the Pytest session is not recognized as interactive."""
     assert not ut.is_interactive_session()
```

Comment:
@marscher I removed the code sample from the docstring because it confused pycharms test configuration. Is there any purpose for this line or did we just miss it in the code review?